recettes/sucrees/galette_des_rois.md

```diff
@@ -0,0 +1,33 @@
+# Galette des rois à la frangipane
+
+---
+
+### Ingrédients
+
+- [ ] 2 pâtes feuilletées
+- [ ] 200g poudre d'amandes
+- [ ] 100g sucre semoule
+- [ ] 2 oeufs
+- [ ] 1 jaune d'oeuf
+- [ ] 100g beurre mou
+- [ ] extrait d'amande amère
+- [ ] 1 fève
+
+---
+
+### Recette
+
+- Pour la frangipane : 
+  - mélanger poudre d'amandes, sucre, oeufs, beurre mou, extrait d'amande amère
+- Pour la galette : 
+  - découper un disque dans chaque pâte feuilletée, disposer les 2 anneaux de pâte restants sur les bords d'un des disques, étaler la frangipane à l'intérieur de ces anneaux
+  - Mettre la fève (au bord), recouvrir du second disque de pâte, et bien souder les bords
+  - Décorer la pâte avec un couteau
```

Comment:
à l'aide d'un couteau to be consistent with below

Reply:
good catch